needle-notion-connector/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Let's put this file in gitignore. Some devs may not be using vscode.

Reply:
removed .vscode from the repo and added it to gitignore